src/components/atoms/OcSearchBar/OcSearchBar.vue

```diff
@@ -257,9 +257,8 @@ export default {
   &-input {
     @extend .uk-input;
 
-    background-color: var(--oc-color-input-bg);
-    border: 1px solid var(--oc-color-input-border);
-    border-radius: 3px;
+    border-radius: 100px;
```

Comment:
25px seems to get the job done for me too, wondering what a sane value would be :D

Reply:
was copied from figma :D but yeah should be enough

Reply:
will remove OcAppSideBar too